WeatherPortal/WeatherPortal/settings.py

```diff
@@ -31,17 +31,18 @@
 
 ALLOWED_HOSTS = ['*']
 
-
 # Application definition
 
 INSTALLED_APPS = [
+    # weatherapp must be first !!! must go before the admin application in INSTALLED_APPS
```

Comment:
czemu?

Reply:
jak weatherapp był na dole INSTALLED_APPS, to opcja zmiany hasła prznosiły nas na adminowa zmiane hasła użytkownika, jak jest na górze, do przenosi nas na właściwą strone, jako że python przetwarza dane od góry do dołu, to wie żeby brać najpierw informacje z weatherapp a potem z  'django.contrib.admin'. Ostatecznie takie rozwiązanie tego problemu znalazlem na stackoverflow i działa :)

Reply:
Spojrzę dokładnie później, ale zawsze dawałem apki na dole i było ok

Reply:
https://stackoverflow.com/questions/447512/how-do-i-override-djangos-administrative-change-password-page

"If you still see the Django's Admin template, you must change the order of your INSTALLED_APPS (e.g if your template is inside some app, this app must appear before django.contrib.admin in INSTALLED_APPS)"

Reply:
No ok, ale pytanie brzmi czemu chcesz nadpisać zmianę hasła po stronie admina? To jest zmiana tylko dla jakichś adminów, pracowników? Bo jeśli nie to powinien być jakichś dodatkowy endpoint do zmiany hasła i tam byłaby używana templatka jakaś Wasza, jak każdy inny endpoint, jest też już coś wbudowanego w Django

https://ccbv.co.uk/projects/Django/4.0/django.contrib.auth.views/PasswordChangeView/
https://ccbv.co.uk/projects/Django/4.0/django.contrib.auth.views/PasswordChangeDoneView/

Są też klasy do resetu hasła, można zobaczyć implementację bo Wy pewnie będziecie robić to podobnie, a może i warto byłoby skorzystać z tego?